.github/workflows/release.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+name: Release
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          path: awm-relayer
+      - name: Set Go version
```

Comment:
Nit: Could you put an extra newline in between the steps here so that it's slightly easier to read?